File: EventServiceApi/EventServiceApi/Models/WasherDead.cs
using EventServiceApi.ViewModels;
using System.ComponentModel.DataAnnotations;

namespace EventServiceApi.Models
{
    public class WasherDead
    {
        [Key]
        public long Id { get; set; }

        [Required]
        [MaxLength(50)]
        public string FirstName { get; set; }

        [MaxLength(50)]
        public string LastName { get; set; }

        [Required]
        [MaxLength(50)]
        public string RequiredPhoneNumber { get; set; }

        [MaxLength(50)]
        public string AdditionalPhoneNumber { get; set; }

        [Required]
        [MaxLength(50)]
        public string City { get; set; }

        [MaxLength(50)]
        public string District { get; set; }


        public WasherDeadViewModel ConvertToViewModel()
        {
            return new WasherDeadViewModel()
            {
                WasherDeadId = Id,
                WasherDeadFirstName = FirstName,
                WasherDeadLastName = LastName,
                WasherDeadRequiredPhoneNumber = RequiredPhoneNumber,
                WasherDeadAdditionalPhoneNumber = AdditionalPhoneNumber,
                WasherDeadCity = City,
                WasherDeadDistrict = District
            };
        }
    }
}

File: app/src/main/java/com/example/tadbir/WasherDead.java
package com.example.tadbir;

import com.google.gson.annotations.SerializedName;

public class WasherDead {
    @SerializedName("Id")
    public long id;
    @SerializedName("FirstName")
    public String firstName;
    @SerializedName("LastName")
    public String lastName;
    @SerializedName("RequiredPhoneNumber")
    public String phoneNumber;
    @SerializedName("AdditionalPhoneNumber")
    public String additionalPhoneNumber;
    @SerializedName("City")
    public String city;
    @SerializedName("District")
    public String district;
}
